_layouts/default.html

```diff
@@ -1,8 +1,52 @@
 <!DOCTYPE html>
 <html lang="en">
-  {% include _head.html %}
-  <body class="home">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      {{ site.title }}
+    </title>
+    <meta name="author" content="{{site.author}}">
+    <meta name="description" content="{{site.description}}">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="/assets/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.rawgit.com/konpa/devicon/89f2f44ba07ea3fff7e561c2142813b278c2d6c6/devicon.min.css">
+  </head>
+  <body class="container">
+    <header class="text-c mt3">
+      <h1 class="mb1">Stitch Fix Open Source</h1>
+      <p>
+        Open Source software is a huge part of our success.  This site is our small contribution back.
```

Comment:
How about *"These projects are our small contribution back"* (or something similar)?  The projects are more useful than this site.  :wink: